src/Validation/Validator.php

```diff
@@ -129,12 +134,42 @@ public function validateParams(array $data): array
 			$fieldsOnly = $this->getValidationReferenceManual($fieldsOnly);
 		}
 
+		// Find refference fields in admin config.
 		$validationReference = $this->getValidationReference($fieldsOnly);
 
+		// Find all req fields.
```

Comment:
```suggestion
		// Find all required fields.
```
Just to keep things verbose